scripts/mkcloud

```diff
@@ -121,6 +121,7 @@ scp="scp $sshopts"
 : ${localreposdir_target:="/repositories"}
 [ -z "$localreposdir_src" ] && localreposdir_target=""
 : ${install_chef_suse_override:=./install-chef-suse.sh}
+: ${cct_tests:=features}
```

Comment:
I'm not sure about the `cct_tests` default name, it does not express what it does, I would rather change that to `base` and adapt the testsuite to provide `rake features:base`. Then it should be easy to combine multiple test categories like `base+storage+ha` and delegate that to respective `rake` tasks.